configure.ac

```diff
@@ -65,11 +65,8 @@ AC_SUBST([LIBS], ["$X11_LIBS $XCOMPOSITE_LIBS $XEXT_LIBS $XFIXES_LIBS \
 AC_SUBST([CPPFLAGS], ["$X11_CFLAGS $XCOMPOSITE_CFLAGS $XEXT_CFLAGS \
     $XFIXES_CFLAGS $XINERAMA_CFLAGS $IMLIB2_CFLAGS $LIBBSD_CFLAGS $CPPFLAGS"])
 
-# Checks for header files.
-AC_CHECK_HEADERS([stdint.h sys/time.h unistd.h])
-
 # Required: Checks for library functions.
-AC_CHECK_FUNCS([getopt_long getsubopt gethostname strdup strerror strndup strtol],,
+AC_CHECK_FUNCS([getopt_long],,
```

Comment:
The compilation will fail if the function or header is not available, so I'd remove the `getopt_long()` check too. This PR will speed up our compilation process a lot.

Reply:
I was thinking the same. scrot isn't chromium or something like there where it takes multiple hours to compile - so pre-checking for functions aren't that useful.

Only reason for keeping `getopt_long` in there was because it's non-standard. But I have no problems removing it as well.